libgloss/arc/hl/README.md

```diff
@@ -0,0 +1,50 @@
+Metaware hostlink IO
```

Comment:
I don't think this Readme is well placed here.

Reply:
Moved one level up as `readme-hostlink.md`.

Reply:
You can make it a more complex document and move it to ABI repo.

Reply:
There are two files in libgloss with similar content: `./m68k/m68k-semi.txt` and `./nios2/nios2-semi.txt`.

I'm not sure where to place this document.